Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/TimePickerSamplePage.xaml

```diff
@@ -3,13 +3,15 @@
 	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
 	  xmlns:local="using:Uno.Gallery"
 	  xmlns:smtx="using:ShowMeTheXAML"
+      xmlns:mobile="http://uno.ui/mobile" 
+      xmlns:not_mobile="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  xmlns:ios="http://uno.ui/ios"
 	  xmlns:android="http://uno.ui/android"
-	  mc:Ignorable="d ios android">
+	  mc:Ignorable="d ios android mobile">
 
-	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	<not_mobile:Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
```

Comment:
You can set the platform-prefix on the `DataTemplate`:
```xml
<...MaterialTemplate>
   <mobile:DataTemplate>....</...>
   <not_mobile:DataTemplate>...</...>
```
This will help avoid the duplicated codes.

Reply:
also while here, you can merge the ios+android:TextBlock on L43+46

Reply:
Ah true, I like that better